### DISCUSSION TITLE
When will the 2.212.1 version be available with Homebrew?

### DISCUSSION THREAD

**Discussion by flyingsofa** [fastlane/fastlane#21078] [Q&A]
I noticed that the 2.212.1 release is available on Github but not yet with [Hombrew](https://formulae.brew.sh/formula/fastlane). 

Is it usually updated at the same time? 

Thanks!

**Comment by aaronksaunders** [comment]
is there some other way to get the 2.212.1 version, I am getting an error logging into apple to push a release that I believe in resolved in 2.212.1
